content/docs/guide/gamemodes/rocket_jump.md

```diff
@@ -14,8 +14,11 @@ Based on the Soldier class from Team Fortress 2, players shoot unidirectional, f
 
 ## History of Rocket Jump
 
-TODO
+The origins of rocket jumping can be traced all the way back to the release of Doom in 1993. Players could sacrifice health to cross gaps not originally intended to be crossed by firing a rocket at a wall nearby and use the outward horizontal velocity emitted by the explosion of the rocket (sometimes called the "splash" or "splash damage") to strafe across a gap. Other games, like Rise of the Triad (1994) and Marathon (1995), also had similar mechanics, leading to the rise of Rocket Jumping as a "Pro Strategy" for beating certain levels fast or showing off in deathmatch. This was refined in Quake, where the verticality offered by the Quake Engine allowed for players to be able to launch themselves upwards using the rocket launcher, skipping parts of levels, finding secrets, or gaining height advantages in classic Deathmatch gamemodes. Quake is argued to be the true beginning of Rocket Jumping as we know it, as it was the most popular early example of rocket jumping. This is also the beginning of players making maps specifically dedicated to practicing rocket jumping.
+
+As Valve built their (new for the time) Goldsrc engine on top of the Quake Engine's foundation, this mechanic was brought into Half-Life 1 and its mods. It should be noted that Half-Life's Tau Cannon functions similarly to a rocket jump, but varies slightly in execution and player damage. Half-Life's rocket jumping was used in similar vein to Quake's rocket jumping within the main game, utilized by speedrunners or deathmatch players, but also began to take on a game mode of its own around this time. WIth the release of the mod Team Fortress Classic, and the inclusion of two player classes, the Demoman and Soldier, rocket jumping began taking a new role outside of deathmatch. Players began creating maps in droves to practice new rocket jumping techniques like pogo, sync-shots, and more. As Valve were developing their sequel the Team Fortress, they took note of the community's interest in such a mechanic, and so designed TF2's Soldier to include rocket jumping as a key part of gameplay for the class. From there, we are brought to today's rocket jumping, and all of the complexities and mechanics it comes with.
```

Comment:
Thanks for the feedback, might if I ask what the exact changes within Mmod are to RJ to describe them in the doc accurately?

Reply:
As a suggestion, maybe the "From there" part could instead be the segue into momentum mods' implementation of RJ. 

Something along the lines of: "From there, Momentum Mod began its' interpretation of the mechanic. Largely derived from TF2 rocket jumping, Momentum Mod RJ explicitly sought to remove the many frustrations present in TF2 rocket jumping, and to standardize the experience due to some of the contentious "tech" that has been developed over the game modes lifetime."

Then perhaps the "contentious" tech that mmod addressed such as wallbugs/bounces could be singled out in the "How Rocket Jump Works" section, instead of the history section (feels more appropriate that way, to me.)